app/components/Instructions/Instructions.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Instructions({ state, }) {
+  function buildInstructionArray(appState) {
```

Comment:
'build' is a bad name. Usually the verbs for creating something are 'get' or 'create'.

Reply:
done